fuzz/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "goblin-fuzz"
+version = "0.0.1"
+authors = ["Automatically generated"]
```

Comment:
You can add yourself to authors here. I can make you Lead Liason and Executive Officer for Goblin Fuzzing if you like :)